src/components/ContentHighlights/ContentHighlightsCardItemsContainer.jsx

```diff
@@ -1,18 +1,14 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { CardGrid } from '@edx/paragon';
-import { camelCaseObject } from '@edx/frontend-platform';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import ContentHighlightCardItem from './ContentHighlightCardItem';
-import { TEST_COURSE_HIGHLIGHTS_DATA } from './data/constants';
+import { generateAboutPageUrl } from './data/constants';
 
-const ContentHighlightsCardItemsContainer = () => {
-  const [highlightCourses] = useState(
-    camelCaseObject(TEST_COURSE_HIGHLIGHTS_DATA)[0]?.highlightedContent,
-  );
-
-  if (!highlightCourses) {
-    return null;
+const ContentHighlightsCardItemsContainer = ({ enterpriseSlug, isLoading, highlightedContent }) => {
+  if (!highlightedContent || highlightedContent?.length === 0) {
+    return <div data-testid="empty-highlighted-content" />;
```

Comment:
Rather than rendering an empty `div` here, I think it'd be worth rendering some sort of reasonable "empty state"  here, even though it theoretically shouldn't happen since you need to create a highlight set with some initial content.

Rendering an `Alert` in this situation would at least indicate something in the UI vs. a blank space, e.g.:

```jsx
<Alert variant="warning">
  There is no highlighted content for this highlight collection.
</Alert>
```